packages/chrono-mongo-datastore/test/factories/task.factory.ts

```diff
@@ -0,0 +1,20 @@
+import { faker } from '@faker-js/faker';
+import { type Task, type TaskMappingBase, TaskStatus } from '@neofinancial/chrono-core';
+import { Factory } from 'fishery';
+
+export const defineTaskFactory = <TaskMapping extends TaskMappingBase>(
+  taskKind: keyof TaskMapping,
+  defaultTaskData: TaskMapping[keyof TaskMapping],
+) =>
+  Factory.define<Task<keyof TaskMapping, TaskMapping[keyof TaskMapping]>>(
+    (): Task<keyof TaskMapping, TaskMapping[keyof TaskMapping]> => {
+      return {
+        id: faker.database.mongodbObjectId(),
+        kind: taskKind,
+        status: faker.helpers.objectValue(TaskStatus),
+        data: defaultTaskData,
+        scheduledAt: faker.date.past(),
+        originalScheduleDate: faker.date.past(),
+      };
+    },
+  );
```

Comment:
turns out i didnt need this. But was kinda a pain making so going to keep it in case we need it